server/src/main/java/org/spine3/server/entity/EntityRepository.java

```diff
@@ -159,6 +159,11 @@ public E load(I id) {
         while (idIterator.hasNext() && recordIterator.hasNext()) {
             final I id = idIterator.next();
             final EntityStorageRecord record = recordIterator.next();
+
+            if (record == null) { // Record is nullable here since RecordStorage::findBulk returns an Iterable that may contain nulls.
```

Comment:
Also, please wrap `{@code Iterable}`. This is important in similar cases in order to produce nice JavaDocs.